drivers/adodb-sqlite3.inc.php

```diff
@@ -596,6 +596,119 @@ function year($fld)
 		$x = "strftime('%Y',$fld)";
 		return $x;
 	}
+	
+	/**
+	* SQLite update for blob 
+	*
+	* SQLite must be a fully prepared statement, so where can either
```

Comment:
Not sure what you mean by _SQLite must be a fully prepared statement_

Reply:
We often write code that is "partially" prepared - for example the function I'm addressing looks like this in other drivers:
````php
UPDATE sessions2 SET sessdata=? WHERE sesskey='dsdfsfdsfsfsfsdfsd'", 
````
This isn't acceptable in SQLite for updating blob data. You have to bind **every** variable, i.e. make it "fully" prepared:
````php
UPDATE sessions2 SET sessdata=? WHERE sesskey=?", 

Thats what I'm addressing

Reply:
Ah OK I get it, didn't realize this (I don't use sqlite much). For the interest of others who like me are not aware of such technicalities, maybe improve the comment like

`SQLite must be a fully prepared statement, i.e. having all of its variables bound` or something like that